knowledge_base/job_with_condition_task/databricks.yml

```diff
@@ -0,0 +1,38 @@
+bundle:
+  name: job_with_condition_task
+
+variables:
+  warehouse_id:
+    lookup:
+      warehouse: "Shared Warehouse"
+
+resources:
+  jobs:
+    example_job:
+      name: "Example job to demonstrate the use of a condition task"
+
+      tasks:
+        - task_key: condition
+          condition_task:
+            left: "{{ job.parameters.[run_sql_task] }}"
```

Comment:
Could we show an example with a task value instead? So we have one Python notebook compute something, return it, and then use it in the conditional?

(Btw we want to support that use case for SQL queries too but don't have that support yet.)

The current example using a job parameter is a bit atypical.

+ @gaborratky-db

Reply:
Could that be an additional example? For this one, I wanted to repro an issue and capture the example.

Reply:
We could have a notebook "process_data" that does some processing and sets a task value "optimize" that determines whether to run `OPTIMIZE <table>` in a subsequent SQL query task:

if `{{ tasks.process_data.values.optimize }} == "true"` => `OPTIMIZE <table>`

and do nothing if false